ui/src/app/utils/access-utils.tsx

```diff
@@ -461,10 +461,19 @@ const withInvalidDateHandling = (date) => {
 export const hasExpired = (expiration: number): boolean =>
   !!expiration && getWholeDaysFromNow(expiration) < 0;
 
-export const isExpiringOrExpired = (expiration: number): boolean =>
-  !!expiration &&
-  getWholeDaysFromNow(expiration) <=
-    serverConfigStore.get().config.accessRenewalLookback;
+export const isExpiringOrExpired = (
+  expiration: number,
+  module: AccessModule
+): boolean => {
+  const moodleModules = [
+    AccessModule.COMPLIANCETRAINING,
+    AccessModule.CTCOMPLIANCETRAINING,
+  ];
+  const lookback = moodleModules.includes(module)
+    ? serverConfigStore.get().config.complianceTrainingRenewalLookback
```

Comment:
Also, consider adding a test case to `access-utils.spec.tsx` or  `access-renewal.spec.tsx` to cover this

Reply:
Added a couple tests to access-utils.spec.tsx. Please let me know if you think any others are needed.

Reply:
Looks reasonable, thanks!